src/routes/v0/queries.rs

```diff
@@ -1,44 +1,45 @@
-use crate::models::post::{PostStreamSorting, ViewerStreamSource};
-use serde::de::Deserializer;
+use serde::de::{self, Deserializer};
 use serde::Deserialize;
+use utoipa::ToSchema;
 
 #[derive(Deserialize)]
 pub struct TagsQuery {
     pub limit_tags: Option<usize>,
     pub limit_taggers: Option<usize>,
 }
 
-#[derive(Deserialize)]
+#[derive(Default, Deserialize, Debug, ToSchema)]
 pub struct PaginationQuery {
```

Comment:
We are using `PaginationQuery` for many more things than the axum handler queries. In fact, `Query` . What about we call this struct simply `Pagination` and move it to `src/types.rs`  or `src/models/types.rs` ? 

Maybe there is other small types around our codebase that could be move into `types.rs` . Enum JsonAction or ScoreAction? I don't know :)